.circleci/config.yml

```diff
@@ -955,8 +955,6 @@ workflows:
           docker_name: op-node
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
           docker_context: .
-          context:
```

Comment:
this context is no longer necessary since we've separated publish from build (also forked builds will now work)